service/src/main/java/com/myhome/controllers/HouseController.java

```diff
@@ -76,6 +85,20 @@
         .orElse(ResponseEntity.notFound().build());
   }
 
+  @Override public ResponseEntity<ListHouseHistoryResponse> getHouseHistory(String houseId,
+      @Valid String memberId) {
+    if (!houseService.getHouseHistory(memberId,houseId).isPresent()) {
```

Comment:
Please do not use `isPresent` if not really needed.
You can work with `.map` operator